### DISCUSSION TITLE
How should I configure the color of the code highlight ?

### DISCUSSION THREAD

**Discussion by gcxfd** [vuepress/core#539] [Q&A] [answered]
How should I configure the color of the code highlight, I want the background of the code highlight to be white? But only the background is set, and the text color of the code highlight is too light

**Comment by meteorlxy** [comment, marked as answer]
Override the code styles in the style file:

https://v2.vuepress.vuejs.org/reference/default-theme/styles.html#style-file

**Comment by Mister-Hope** [comment]
Also, you may check third party theme to see if they support customizing it.
